### DISCUSSION TITLE
inconsistent <a> styling

### DISCUSSION THREAD

**Discussion by panva** [npm/feedback#149] [Website] [answered]
The css applied to package view when markdown is rendered is inconsistent when it comes to links. They use different font-weight depending on where they are placed - paragraph, list, etc.

Screenshots taken from https://www.npmjs.com/package/jose

<img width="754" alt="Screenshot 2021-01-12 at 15 42 13" src="https://user-images.githubusercontent.com/241506/104329193-f850ec00-54ec-11eb-99f5-b75b29f61f4a.png">
<img width="699" alt="Screenshot 2021-01-12 at 15 41 42" src="https://user-images.githubusercontent.com/241506/104329203-fa1aaf80-54ec-11eb-9519-247d098ab068.png">

**Comment by MylesBorins** [comment, marked as answer]
hey @panva thanks for opening this discussion. I've made a ticket with our engineering team and will let you know if we plan to take next steps.

**Reply by MylesBorins** [reply]
hey @panva unfortunately at this time the team is not going to be able to prioritize fixing this. We have the issue tracked in the backlog and may revisit in the future. Thank you for reporting.
